packages-web/maps/src/components/LeafletMap.ts

```diff
@@ -224,7 +224,9 @@ export class LeafletMap extends Component<LeafletMapProps, LeafletMapState> {
 
     private createMarker = (location: Location): Marker => {
         const { latitude, longitude, url } = location;
-        const marker = new Marker([Number(latitude), Number(longitude)]);
+        const marker = new Marker([Number(latitude), Number(longitude)], {
+            GUID: location.mxObject!.getGuid()
```

Comment:
what if `location.mxObject` doesn't exists ?

Reply:
Onclick actions are only available with dynamic data (Database, Context, Nanoflow or Microflow). This means this object will never be null

Reply:
Done